BuildTools.psm1

```diff
@@ -996,4 +996,106 @@ function Get-GitTimeStampForScript($script) {
     } finally {
         Pop-Location
     }
-}
\ No newline at end of file
+}
+
+# Notice the year is incomplete.  We fill it in Add-Copyright below.
+$copyrightTemplate = @"
+Copyright 20 Google
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not
+use this file except in compliance with the License. You may obtain a copy of
+the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations under
+the License.
+"@

```

Comment:
should we add an empty line, so that automatically we get a space between copyright and usings?

Reply:
Also, we should probably consider using [stylecop](https://marketplace.visualstudio.com/items?itemName=ChrisDahlberg.StyleCop) and define a rule that will require this copyright.